changelog.txt

```diff
@@ -4,6 +4,13 @@ Changes
 
 The **signac-dashboard** package follows `semantic versioning <https://semver.org/>`_.
 
+next
+====
+
```

Comment:
Sphinx requires a second level heading here. You can't skip from first-level `=====` to third-level `+++++` or else it renders incorrectly.
```suggestion
next
====

next
----

```